diffsync/diff.py

```diff
@@ -115,6 +116,13 @@ def summary(self) -> Mapping[Text, int]:
             child_summary = child.summary()
             for key in summary:
                 summary[key] += child_summary[key]
+        summary[DiffSyncActions.SKIP] = (
+            self.models_processed
+            - summary[DiffSyncActions.CREATE]
+            - 2 * summary[DiffSyncActions.UPDATE]
```

Comment:
what is the reason for doubling the update and no.change types?

Reply:
If ther is an action "update" or "no-change" it means the node was in source and destination so the `DiffSyncDiffer` counts it twice. On "create" or "delete" it is only existing in one side.

The model_processed logic was already inplemented in hte DiffSyncDiffer class and I did not want to change the logic of it

Reply:
I think it would be good to get a little comment into the code that explains this, because to me it also wasn't immediately clear.